src/containers/edit-profile/professional-info-tab/about-tutor-accordion/AboutTutorAccordion.tsx

```diff
@@ -1,22 +1,34 @@
-import Accordions from '~/components/accordion/Accordions'
+import { FC, useState } from 'react'
+
+import { useTranslation } from 'react-i18next'
+
+import {
+  AccordionItem,
+  ProfessionalBlock,
+  TypographyVariantEnum
+} from '~/types'
+
 import ExpandMore from '@mui/icons-material/ExpandMore'
+
+import Accordions from '~/components/accordion/Accordions'
 import AppTextArea from '~/components/app-text-area/AppTextArea'
-import { useState } from 'react'
-import useForm from '~/hooks/use-form'
-import { useTranslation } from 'react-i18next'
+
 import { styles } from '~/containers/edit-profile/professional-info-tab/about-tutor-accordion/AboutTutorAccordion.styles'
-import { initialFormValues } from '~/containers/edit-profile/professional-info-tab/about-tutor-accordion/AboutTutorAccordion.constants'
-import { AccordionItem, TypographyVariantEnum } from '~/types'
 
-const AboutTutorAccordion = () => {
+interface AboutTutorAccordionProps {
+  data: ProfessionalBlock
+  handleInputChange: (
+    key: keyof ProfessionalBlock
+  ) => (event: React.ChangeEvent<HTMLInputElement>) => void
```

Comment:
Use following type from `src/types/common.types.ts`
```jsx
export type UseFormEventHandler<Fields, Event> = (
  key: keyof Fields
) => (event: Event) => void
```

Reply:
Didn't know about this type, thank you, fixed